codeface_extraction/extractions.py

```diff
@@ -633,3 +636,19 @@ def fix_characters_in_string(text):
     new_text = u"".join(ch if unicodedata.category(ch)[0] != "C" else " " for ch in new_text.decode("unicode-escape"))
 
     return new_text
+
+def gen_range_directory(range_number, start_revision, end_revision):
```

Comment:
Is there a reason why you do not import the function from Codeface? As far as I understand the code, this should work out nicely.

Reply:
Good point. I re-worked the solution for this issue step by step and did not recognize afterwards that the outcome is exactly the same as what the function from Codeface does. So I can easily incorporate this while rebasing this PR.